### PR TITLE
Add implementation docs (revert the revert)

### PR DESCRIPTION
**Pull request by maya**
This re-adds implementation docs for max-width text, lists, buttons, and grids.

Original PR: #673.

This reverts commit d63bb18d588e5578834f332d587a21a649c02189, reversing
changes made to d3254536359678cca06c58f4841f9378f4bf4fa7.

This also adds in `<wbr>` tags so the pages display correctly (tested changes on iPhone 5 & 6).

### CODE REVIEW

**Comment by alexose** [comment]
:+1: 

Lots of good stuff here!

File: _visual/typography.md [hunk 1]
```diff
@@ -998,11 +1002,13 @@ order: 01
...
   </button>
   <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
+  <h4 class="usa-heading">Implementation</h4>
+  <p>Lists are styled by default. For unstyled lists, use either the <code>usa-unstyled-list</code> class or unstyled list mixin: <code>@include unstyled-list;</code>. Both are located in <code>assets/_scss/core/<wbr>utilities.scss</code>.</p>
     <h4 class="usa-heading">Usability</h4>
     <h5>When to use</h5>
     <ul class="usa-content-list">
-        <li>Use an ordered list when you need to display text in some ranking, hierarchy or series of steps.</li>
-        <li>Use unordered lists display text in no specific order.</li>
+      <li>Use an ordered list when you need to display text in some ranking, hierarchy or series of steps.</li>
```
Comment (awfrancisco) [inline _visual/typography.md]: Should be "ranking, hierarchy, or series"

File: _visual/typography.md [hunk 2]
```diff
@@ -998,11 +1002,13 @@ order: 01
...
+      <li>Use unordered lists display text in no specific order.</li>
```
Comment (awfrancisco) [inline _visual/typography.md]: "...lists **to** display text..."

**Comment by awfrancisco** [comment]
Added two small edits, otherwise good to go.

**Comment by awfrancisco** [comment]
I'm going to wait to copy edit this section until this PR is merged.

**Pull request merged**